dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/NettyRemotingClient.java

```diff
@@ -160,6 +161,7 @@ private void start() {
             .option(ChannelOption.TCP_NODELAY, clientConfig.isTcpNoDelay())
             .option(ChannelOption.SO_SNDBUF, clientConfig.getSendBufferSize())
             .option(ChannelOption.SO_RCVBUF, clientConfig.getReceiveBufferSize())
+            .option(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT)
```

Comment:
The default ALLOCATOR is pooled, please refer to ByteBufUtil

Reply:
Thx, you are right